tests/e2e/vsphere.go

```diff
@@ -1411,3 +1411,72 @@ func getAggregatedSnapshotCapacityInMb(e2eVSphere vSphere, volHandle string) int
 
 	return queryResult.Volumes[0].BackingObjectDetails.(*cnstypes.CnsBlockBackingDetails).AggregatedSnapshotCapacityInMb
 }
+
+// renameDs
+func (vs *vSphere) renameDs(ctx context.Context, datastoreName string,
```

Comment:
renameDs and renameDsInParallel , can be combined ?

Reply:
Yes, I have made the changes to the util accordingly.